### ISSUE TITLE
Web client hard codes http prefix

### ISSUE THREAD

**Issue by armaros** [athensresearch/athens#2017]
**Problem**
Using 
https://athensresearch.github.io/athens/
Attempt to add a self hosted backend. 
The web client will attempt a health check on the self hosted backend.
This line will prepend "http://" to the url
https://github.com/athensresearch/athens/blob/06cedc1cd360b537b451ad20768098a3c9137d49/src/cljs/athens/effects.cljs#L158

The web client will reject the connection with the following error:
```
Mixed Content: The page at 'https://athensresearch.github.io/athens/#/' was loaded over HTTPS, but requested an insecure resource 'http://https//athens.xxx.com/health-check'. This request has been blocked; the content must be served over HTTPS.
```

**Comment by armaros** [comment]
Is appears that simply changing the hardcoding to https in the js code will simply result in hitting an error attempting to construct an insecure web sockets connection latter on.

**Comment by armaros** [comment]
![Screenshot 2022-02-06 222201](https://user-images.githubusercontent.com/2020499/152720298-a3492ed5-a585-4e47-be7b-d22915427a73.png)

This is the error from the browser if you attempt to change the hardcoding to https

**Comment by filipesilva** [comment]
Fixed via #2022
